src/extension.ts

```diff
@@ -92,9 +92,19 @@ export async function activate(context: vscode.ExtensionContext) {
 		// Register tab change event
 		vscode.window.tabGroups.onDidChangeTabs(changeEvent => {
 			try {
-				const isTabEvent = changeEvent.changed.length == 0 && (changeEvent.opened.length > 0 || changeEvent.closed.length > 0)				
+				let fileChanged = false;
+                if(changeEvent.changed.length > 0){
+                    const oldPath = CodeSyncState.get(CODESYNC_STATES.ACTIVE_FILE_PATH);
```

Comment:
We should not continue if we don't have anything saved in this variable in first iteration